spec/spec_helper.rb

```diff
@@ -286,6 +286,9 @@ def json_document model, record_name
 end
 
 AUTH_USERS = {
+  superuser: {
+    email: 'zeus@twelve-labors.org', first: 'Zeus', perm: 'a:administration'
```

Comment:
Should the permissions on this new user be for the `labors` project instead of `administration`?

Reply:
`administration` makes the user a superuser.

```ruby
module Etna
  class User
    # …
    def is_superuser? project=nil
      has_roles(:administration, :admin)
    end
    # …
  end
end
```

https://github.com/mountetna/etna/blob/089c179a66f3b451a82cea8ddec133f738f6948e/lib/etna/user.rb#L55-L57

If we changed it to `labors`, the user would only have admin access on the Labors project.

Reply:
Ah interesting. Thanks for clarifying!